packages/server/src/modules/retail-location/retail-location.resolver.ts

```diff
@@ -167,26 +167,24 @@ export class RetailLocationResolver {
       message: "You do not have permission to view these reservations.",
     });
 
-    const getTotalBooksWithProblems = async () => ({
-      totalBooksWithProblems: await this.prisma.bookCopy.count({
-        where: {
-          book: {
-            retailLocationId,
-          },
-          problems: {
-            some: {
-              resolvedAt: null,
-            },
-          },
-          returnedAt: {
-            not: null,
+    const getTotalBooksWithProblems = this.prisma.bookCopy.count({
+      where: {
+        book: {
+          retailLocationId,
+        },
+        problems: {
+          some: {
+            resolvedAt: null,
           },
         },
-      }),
+        returnedAt: {
+          not: null,
+        },
+      },
     });
 
-    const getTotalPresentBooks = async () => ({
-      totalPresentBooks: await this.prisma.bookCopy.count({
+    const getTotalPresentBooks = async () =>
```

Comment:
Why did you kept the `async` here and in the following cases?
Just get the `this.prisma.xxx` result as in the first one you fixed and some of the following ones

Reply:
Because I remembered that you told me to leave async so you can recognize it as an async function.
I'll remove that

Reply:
You got it partially wrong: why didn't you just remove the arrow function entirely as you did in some other instances?